nsxt/resource_nsxt_policy_project.go

```diff
@@ -117,28 +117,21 @@ func resourceNsxtPolicyProject() *schema.Resource {
 				MaxItems:    1,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
-						"span_reference": {
+						"default_span_path": {
+							Type:         schema.TypeString,
+							Description:  "Policy path of the Cluster based default Span object of type NetworkSpan",
+							ValidateFunc: validatePolicyPath(),
+							Optional:     true,
+							Computed:     true,
+						},
+						"non_default_span_paths": {
 							Type:        schema.TypeList,
 							Description: "List of Span object references available with the project for TGW consumption",
```

Comment:
This description is not correct anymore (you can just use the same description from the doc).